src/form/birth/optional-fields.ts

```diff
@@ -83,20 +83,13 @@ export const multipleBirth: SerializedFormField = {
       expression: '!values.detailsExist'
     }
   ],
+  maxLength:2,
   required: false,
   initialValue: '',
-  validator: [
-    {
-      operation: 'greaterThanZero'
-    },
-    {
-      operation: 'maxLength',
-      parameters: [2]
-    }
-  ],
+  validator: [],
   mapping: getFieldMapping(
     'multipleBirth',
     certificateHandlebars.multipleBirth
   ),
-  inputFieldWidth: '64px'
+  inputFieldWidth: '64px',
```

Comment:
Do you have a prettier plugin installed in your code editor? I think it should remove the trailing comma automatically